File: vaip-core/src/cleanup.cpp
/*
 * Copyright (C) 2023 - 2025 Advanced Micro Devices, Inc. All rights reserved.
 * Licensed under the MIT License.
 */
#include "./cleanup.hpp"
namespace vaip_core {
void deinitialize_onnxruntime_morphizen_ep() { cleanup_all(); }
} // namespace vaip_core

File: tools/rgp_parser/cleanup.py
#!/usr/bin/env python3
#
# Copyright (C) 2026 Advanced Micro Devices, Inc. All rights reserved.
# Licensed under the MIT License.
#
"""Entry point: stop the RGP capture listeners.

python cleanup.py   # kill Radeon Developer Service/Panel
"""

from rgp_parser.cleanup import main

if __name__ == "__main__":
    raise SystemExit(main())
